server/player/player.go

```diff
@@ -367,6 +367,18 @@ func (p *Player) Transfer(address string) (err error) {
 	return
 }
 
+// SendSound send play a sound to the player. If sound to have in vanilla or custom resource pack will be not work
+func (p *Player) SendSound(soundName string, volume, pitch float32) (err error) {
+	p.session().SendSound(soundName, p.Position(), volume, pitch)
+	return
+}
+
+// StopSound send stop play a sound by sound name while sound playing to player
+func (p *Player) StopSound(soundName string, stopAll bool) (err error) {
```

Comment:
I think it would be better to have a `StopSound(soundName string)` and a `StopAllSounds()` since the `soundName` parameter seems useless when `stopAll` is true.

Reply:
OK, I refactor to be a StopSound(soundName string) and a StopAllSounds()